src/nav/NavigationService.js

```diff
@@ -16,19 +16,23 @@ const appContainerRef = React.createRef<
       NavigationContainerProps<{ ... }, NavigationState>>>
 >();
 
-const getState = (): NavigationState => {
+const assertReady = () => {
```

Comment:
Hmm -- can we turn this from `export default { foo, bar }` style to `export const foo = …; export const bar = …` style?

The main nice thing that'd give us is that it's manifest at the definition whether it's going to be exported or not -- which helps for understanding the intended interface.

A small bonus is that if you hit Shift+F12 in VS Code on one of these definitions now, to find where it's used, it'll only show you references within the module. Not *too* bad, because you can go and hit Shift+F12 again over at one of those references -- the `export default` -- and find all the other places it's used. But it'd be nicer to be able to see that in one hop; and it's also potentially misleading to see references only within the module, as it's not necessarily obvious that's what's happening.

To keep the call sites unchanged like `NavigationService.dispatch`, I think all that's needed is to say `import * as NavigationService` rather than `import NavigationService`. See e.g. 3a6abb39b42ba05511fbb2fce4dc076717f68f45, where I made a similar change for `exampleData`.